### ISSUE TITLE
This configuration section cannot be used at this path.

### ISSUE THREAD

**Issue by dotnetfreak** [MohammadYounes/OWIN-MixedAuth#19]
Try as I might I can not get away from a 500.19 Error.  

"This configuration section cannot be used at this path. This happens when the section is locked at a parent level. Locking is either by default (overrideModeDefault="Deny"), or set explicitly by a location tag with overrideMode="Deny" or the legacy allowOverride="false"."

I am using VS2015RC and IISExpress on Windows 7.  
I have updated the applicationhost.config settings to 
```xml
<sectionGroup name="security">
  <sectionGroup name="authentication">
    <section name="windowsAuthentication" overrideModeDefault="Allow" />
  </sectionGroup>
</sectionGroup>
```
I have restarted IISEpress, and my PC.  I still receive the error, using both your example code and when I try to implement it in my project.

**Comment by MohammadYounes** [comment]
Strange!

Check if there is a location override for your app, for example:
```xml
    <location path="Default Web Site" overrideMode="Deny">
        <system.webServer>
            ...
        </system.webServer>
    </location>
```

or `allowOverride="false"` on any parent section.

**Comment by dotnetfreak** [comment]
I managed to get it to work, by enabling "Windows Authentication" in the "Development Server" section of the project properties, and removing the <location> section from the web.config.  Thanks for your help...I was half way through hacking together my own implementation when I found yours (which is considerably more complete, as clearly is your understanding of the owin structure than min), when I came across your solution.  Thus far it works like a charm..next step...implement repositories to use my own table structures.  Cheers.
